routes/index.js

```diff
@@ -87,10 +87,12 @@ router.get('/', (req, res) => {
         "orders": "GET or POST api/v1/orders",
         "order": "GET, PUT or DELETE api/v1/orders/:id",
         "orderBody": {
-            "id": "INT: PUT only",
-            "customer_user_id": "INTEGER",
-            "payment_type_id": "INTEGER",
-            "order_date": "TEXT",
+            "id": "INT",
+            "product_id": "INT: required for POST",
+            "quantity": "INT: optional for POST/PUT, defaults to 1",
```

Comment:
I'm not sure I understand how to update quantity with a put to orders. Let's discuss!